src/service/icon/complex.ts

```diff
@@ -0,0 +1,19 @@
+export class Complex {
+  r: number;
+  i: number;
```

Comment:
What is `r` and `i`? Change naming

Reply:
r - real
i - imaginary
Standard math naming of complex number elements
Ok I'll rename